xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -469,7 +469,11 @@ void CGUIFontTTF::DrawTextInternal(CGraphicContext& context,
         break;
       cursorX += ch->m_advance;
     }
+
+    // Reserve vertices space in vector: 4 * num glyphs
+    tempVertices->reserve(glyphs.size() * 4);
```

Comment:
What does the `4` indicate?

Reply:
Better?
`// Reserve vector space: 4 vertices for each glyph`

Reply:
**4** vertixies for each glyph

Reply:
I'm a big fan of constexpr human readable definitions.

Reply:
It looks good to me but I add it in separate commits since the improvements are probably not necessary backport to Matrix.